go/felix/usagerep/usagerep.go

```diff
@@ -54,6 +50,19 @@ func PeriodicallyReportUsage(interval time.Duration, hostname, clusterGUID, clus
 	}
 }
 
+func calculateInitialDelay(numHosts int) time.Duration {
+	// To avoid thundering herd, inject some startup jitter.
+	if numHosts <= 0 {
+		// Intn requires a positive argument so clamp numHosts to be >= 1.
+		numHosts = 1
+	}
+	initialJitter := time.Duration(rand.Intn(numHosts*1000)) * time.Millisecond
```

Comment:
Presumably the range of a Go integer is large enough that we don't need to worry about numHosts*1000 overflowing?